src/tests/libecalc/output/results/test_ltp.py

```diff
@@ -390,3 +393,14 @@ def test_total_oil_loaded_old_method():
     # Verify that total oil loaded/stored is the same if only loading is specified,
     # compared to a model with both loading and storage.
     assert loaded_and_stored_oil_loading_and_storage == loaded_and_stored_oil_loading_only
+
+
+def test_power_from_shore():
+    """Test power from shore output for LTP export."""
+    [
+        datetime(2027, 1, 1),
+        datetime(2028, 1, 1),
+    ]
+    regularity = 0.2
+
+    ltp_pfs_yaml_factory(regularity=regularity)
```

Comment:
no assert...what is being tested here?

Reply:
The test is not complete yet:-) Will do.

Reply:
Test is done now